e2e/deployment/input/deployment_auto0.nomad

```diff
@@ -0,0 +1,52 @@
+job "deployment_auto.nomad" {
```

Comment:
The job name doesn't need to contain `.nomad` - and it would be nice to add a comment highlighting the significant bits and diff between tasks and files.

Reply:
I know! I got in the habit of doing that so I can up arrow s/run/status/ :) maybe taking lazy too far. I'll add some comments

Reply:
that's a trick i should use myself :)